packages/shared/src/services/trackjs.ts

```diff
@@ -1,101 +0,0 @@
-/*
-    Responses that should be ignored upon receiving errors.
-    We still log them, but having an error inside of these
-    will not break the queue and send them to trackjs.
-    This will look for predefined `ignored_responses_in_trackjs` from GTM, if
-    there is none, then it just does not filter out any response.
-*/
-declare global {
-    interface Window {
-        ignored_responses_in_trackjs: [];
-        TrackJS: { console: { log: (arg0: unknown[]) => void }; track: (arg0: object) => void };
-    }
-}
-
-const getIgnoredResponseKeywords = () => {
-    return window.ignored_responses_in_trackjs || [];
-};
```

Comment:
I think we still need this. Trackjs is still being used. We are just ignoring the BE error here.